scbs/MethylVI_batch.ipynb

```diff
@@ -0,0 +1,468 @@
+{
```

Comment:
<p>jointly = one model but treat both seperately not add both and get one output? Maybe write "we'll jointly model both".</p><p>How would <code>modalities={"categorical_covariate_keys": "mCG"}</code>look like for two different fields. Please put in the comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scvi-tutorials/pull/363/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Maybe you post here the code if both have different categories and then I can advise what's the minimum information required.

Reply:
@canergen what's the exact scenario you're thinking of?

I think I misunderstood your original comment, and I'm not sure what you're asking (was `if mCH and mCH` above a typo?). For example, by two different fields did you mean:

1. I have two categorical covariate fields, one if which lives in `mdata['mCG'].obs` and the other of which lives in `mdata['mCH'].obs`? In this case I believe the `MuDataCategoricalJointObsField` used in the `setup_mudata` function doesn't support this use case, and we would ask users to consolidate their categorical variables into one modality's `.obs` field. 
2. Alternatively, do you mean one argument to the `setup_mudata` function is stored in one modality (e.g. `batch_key` is in `mCG`) while the other is in a different field (e.g. `categorical_covariate_keys` is in `mCH`)? This case should be straightforward, and I can modify the notebook with corresponding instructions.